Project.toml

```diff
@@ -6,8 +6,11 @@ version = "0.2.0"
 [deps]
 Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
 ElasticArrays = "fdbdab4c-e67f-52f5-8c3f-e7b388dad3d4"
+FastIceTools = "14532042-4700-46e0-8dec-55d517bc1ae0"
+FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
```

Comment:
Same here, seems it's the "user" script that needs those deps, not the FastIce project.